### PR TITLE
Fixes around backticks

### PR DESCRIPTION
**Pull request by svick**
I have searched the docs for uninterrupted sequences of letter-backtick-letter and fixed those that I though were wrong. That means:

* Adding a space before and after code.
* Moving backtick that's incorrectly in the middle of a word.
* Other fixes.

### CODE REVIEW

File: docs/framework/tools/al-exe-assembly-linker.md [hunk 1]
```diff
@@ -48,7 +48,7 @@ al sources options
...
... and accessed from managed code in the assembly. You can also do this by using the **/linkresource** compiler option. For more information, see [/linkresource (C# Compiler Options)](~/docs/csharp/language-reference/compiler-options/linkresource-compiler-option.md).<br /><br /> The `name` parameter is an internal identifier for the resource. The `target` parameter specifies a path and file name into which Al.exe copies the `file`*.* After copying, Al.exe compiles `target` into an assembly. By default, resources are public in the assembly (visible to other assemblies). Specifying `private` makes the resource not visible to other assemblies.<br /><br /> If `file` is a .NET Framework resource file created, for example, by the Resource File Generator (Resgen.exe) or in the development environment, it can be accessed with members in the <xref:System.Resources> namespace. For more information, see <xref:System.Resources.ResourceManager>. For all other resources, use the `GetManifestResource` * methods in the <xref:System.Reflection.Assembly> class to access the resource at run time.<br /><br /> If only resource files are passed to Al.exe, the output file is a satellite resource assembly.|
```
Comment (GuardRex) [inline docs/framework/tools/al-exe-assembly-linker.md]: @svick I see a few little things on this topic not directly relevant to your PR. I'll get 'um ... I'm looking for some small fill-in thingies to do before I go. I don't want to hold this set of changes up. I'll work it on:  https://github.com/dotnet/docs/issues/2739

File: docs/framework/tools/ildasm-exe-il-disassembler.md [hunk 2]
```diff
@@ -76,7 +76,7 @@ ildasm [options] [PEfilename] [options]
...
+|**/item:** *class*[**::** *member*[`(`*sig*]]|Disassembles the following depending upon the argument supplied:<br /><br /> -   Disassembles the specified *class*.<br />-   Disassembles the specified `member` of the *class*.<br />-   Disassembles the `member` of the *class* with the specified signature *sig*. The format of *sig* is:<br />     [`instance`] `returnType`(`parameterType1`, `parameterType2`, …, `parameterTypeN`)<br />     **Note** In the .NET Framework versions 1.0 and 1.1, `sig` must be followed by a closing parenthesis: (`sig`). Starting with the Net Framework 2.0 the closing parenthesis must be omitted: (`sig`.|
```
Comment (GuardRex) [inline docs/framework/tools/ildasm-exe-il-disassembler.md]: Same here ... I plan to have a follow-up look at this later in the week on: https://github.com/dotnet/docs/issues/2739

**Review by rpetrusha — APPROVED** [review]
Thanks for making all of these changes, @svick! We'll merge after the update to live PR builds and is merged.

**Review by guardrex — APPROVED** [review]
@rpetrusha Ooops ... I thought I'd save u some time by reviewing. I'm opening a pair of side issues for some things to fix not directly related here, but I reached the same result u did on the review.

File: docs/framework/data/adonet/ef/string-functions.md [hunk 3]
```diff
@@ -35,18 +35,18 @@ The .NET Framework Data Provider for SQL Server (SqlClient) provides `String` fu
...
+|`REPLACE(` `expression1`, `expression2`, `expression3` `)`|Repeats a character expression for a specified number of times.<br /><br /> **Arguments**<br /><br /> `expression1`: The string expression to be searched. string_expression1 can be of a Unicode or an ASCII String type.<br /><br /> `expression2`:The substring to be found. string_expression2 can be of a Unicode or an ASCII String type.<br /><br /> `expression3`; The replacement string. string_expression3 can be of a Unicode or an ASCII String type.<br /><br /> **Example**<br /><br /> `SqlServer.REPLACE('aabbcc', 'bc', 'zz')`|  
+|`REPLICATE(` `char_expression`, `int_expression` `)`|Repeats a character expression for a specified number of times.<br /><br /> **Arguments**<br /><br /> `char_expression`: A Unicode or ASCII `String` type.<br /><br /> `int_expression`: `Int64` (not supported in SQL Server 2000) or `Int32`.<br /><br /> **Return Value**<br /><br /> A Unicode or ASCII `String` type.<br /><br /> **Example**<br /><br /> `SqlServer.REPLICATE('aa',2)`|
```
Comment (svick) [inline docs/framework/data/adonet/ef/string-functions.md]: @GuardRex Done.

**Comment by GuardRex** [comment]
Thanks @svick.

**Pull request merged**